web/index.html

```diff
@@ -3,6 +3,17 @@
 <head>
     <meta charset="utf-8" />
     <title>Expensify.cash</title>
+    <meta name="description" content="Chat and payments are now the same thing. Launching Summer 2021, join the waitlist to be first in line!">
+    <!-- Twitter -->
```

Comment:
I think we can get rid of these comments as it's clear what the tags below are for (they say Twitter).